Jetcaster/wear/src/main/java/com/example/jetcaster/ui/JetcasterNavController.kt

```diff
@@ -45,6 +53,21 @@ public object LatestEpisodes : NavigationScreens("latestEpisodes") {
     public fun destination(): String = navRoute
 }
 
+public object PodcastDetails : NavigationScreens("podcast?podcastUri={podcastUri}") {
+    public const val podcastUri: String = "podcastUri"
+    public fun destination(podcastUri: String): String {
+        val encodedUri = Uri.encode(podcastUri)
+        return "podcast?podcastUri=$encodedUri"
```

Comment:
Since you have a constant declared above you can use it in the returned String.

```suggestion
        return "podcast?$podcastUri=$encodedUri"
```

Reply:
thanks, done